Packs/CommonScripts/Scripts/ListUsedDockersImages/ListUsedDockersImages.py

```diff
@@ -0,0 +1,194 @@
+"""ListUsedDockerImages Script for Cortex XSOAR (aka Demisto)
+
+This is a script that list all the dockers images that are in ues in the integrations and/or automations
+
+"""
+
+import demistomock as demisto
+from CommonServerPython import *
+
+import traceback
+import json
+
+'''REST API HTTP COMMANDS'''
+POST_COMMAND = "POST"
+
+'''REST API CALL BODY'''
+REQUEST_INTEGRATION_SEARCH_BODY = "{\"size\":500}"
+
+'''REST API HTTP ENTRY POINTS'''
+SETTING_INTEGRATION_SEARCH = "/settings/integration/search"
+AUTOMATION_SEARCH = "/automation/search"
+
+'''Constants used to filter out the result'''
+CONFIGURATIONS = 'configurations'
+INTEGRATION_SCRIPT = 'integrationScript'
+
+PYTHON_SCRIPT = 'python'
+POWERSHELL_SCRIPT = 'powershell'
+
+INTEGRATION_SCRIPT_TYPE = 'type'
+INTEGRATION_ID = 'id'
+
+DOCKER_IMAGE = 'dockerImage'
+DEFAULT_DOCKER_IMAGE = 'Default Docker Image'
+
+SCRIPT_TYPE = 'type'
+SCRIPT_ID = 'id'
+SCRIPTS = 'scripts'
+
+MAX_PER_DOCKER = 5
+
+
+''' STANDALONE FUNCTION '''
+
+
+def extract_dockers_from_integration_search_result(content: str) -> dict:
+    """Returns a simple python dict of used dockerImages by integration
+
+    :type content: ``str``
+    :param content: string representation for the /settings/integrations/search API result response
+
+    :return: dict as {"integration_id":"dockername"}
+    :rtype: ``dict``
+
+    """
+    json_content = json.loads(content)
+    dockers = {}
+    for conf in json_content[CONFIGURATIONS]:
+        if (INTEGRATION_SCRIPT not in conf) or (conf[INTEGRATION_SCRIPT] is None):
+            continue
+        else:
+            if INTEGRATION_SCRIPT_TYPE in conf[INTEGRATION_SCRIPT] and conf[INTEGRATION_SCRIPT][SCRIPT_TYPE] in (
+                    PYTHON_SCRIPT, POWERSHELL_SCRIPT):
+                if DOCKER_IMAGE not in conf[INTEGRATION_SCRIPT] or conf[INTEGRATION_SCRIPT][DOCKER_IMAGE] in (None, ''):
+                    docker_image = 'Default Image Name'
+                else:
+                    docker_image = conf[INTEGRATION_SCRIPT][DOCKER_IMAGE]
+                dockers[conf[INTEGRATION_ID]] = docker_image
```

Comment:
check if you can index integration display name and not the integration id.
in case of automation, it should be name